Translation for PT-BR

fclesio opened the pull request:
@VictorGarritano or @haneybarg can you guys perform a review in this one? I didn't translate some terms that we do not have in PT-BR or terms that mean substantives (_e.g._ Reformer, Transformer, BERT, etc).

I've included some italics for foreign terms according to ABNT for facilitating the reading.

fclesio committed:
Translation for PT-BR

omarsar commented on _posts/2020-01-19-NLP_Newsletter[PT-BR]_Reformer,_DeepMath,_ELECTRA,_TinyB.md:
```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: "NLP Newsletter: Reformer, DeepMath, ELECTRA, TinyBERT para busca, VizSeq, Open-Sourcing ML,…"
+author: billy_rick
```
Hi @fclesio you may want the change the author name to the one you provided: fclesio

fclesio replied:
I've included in the credits.

omarsar commented:
@fclesio For terms that you cannot find a translation for, it's okay to leave them as they are. Thanks.

fclesio commented:
Can you merge?

No worries. Thanks a lot!

omarsar commented:
@fclesio I left something for review. If you would like your authorship info to appear in the post. You may want to change the `author` field to your username: `fclesio`

Let me know your choice and then I will merge the PR. Thank you! :)

fclesio committed:
Include credits

fclesio commented:
If it's not possible to include 2 authors, just let your name and merge. 

Thanks once again.

omarsar committed:
updated author information

I removed my authorship information. It's okay just to include yours! Thanks again for doing this. I appreciate the effort. :)

omarsar reviewed (approved):
I updated it to just include your authorship information. It's all fine with me. :) Thanks again for doing this. I appreciate the effort. :)
